src/components/TextField/TextField.scss

```diff
@@ -231,10 +231,6 @@ $stacking-order: (
   &:disabled {
     cursor: default;
   }
-
-  &.ClearButton-hidden {
-    @include visually-hidden;
-  }
```

Comment:
Is there a reason we need to hide the button rather than remove it? We could always add `aria-hidden` if so.